app/src/main/java/com/android/voyageur/ui/overview/AddTrip.kt

```diff
@@ -187,18 +194,8 @@ fun AddTripScreen(
                     .toSet()
                     .toList(),
             locations =
-                locations.split(";").map { locationString ->
-                  val parts = locationString.split(",").map { it.trim() }
-                  if (parts.size >= 2) {
-                    Location(
-                        country = parts[0],
-                        city = parts[1],
-                        county = parts.getOrNull(2),
-                        zip = parts.getOrNull(3))
-                  } else {
-                    Location(country = "Unknown", city = "Unknown", county = null, zip = null)
-                  }
-                },
+                selectedPlace?.let { listOf(Location(address = it.place.displayName ?: "")) }
```

Comment:
We should use the CustomPlace class to store the location, the Location class is deprecated as it was created at the beginning of the project. This causes loss of information since we do not only fetch the display name when searching for places (for example we also fetch the latitude and longitude which might be useful if we want to show different trips on map)

Reply:
I adapted both Trip and Activity objects to use Place in order to store the location, as we don't need to save the photos in firebase. Let me know what do you think.